firecrest/types.py

```diff
@@ -0,0 +1,140 @@
+"""Types returned by the API.
+
+See also: https://firecrest-api.cscs.ch
+"""
+from __future__ import annotations
+
+import sys
+from typing import Any
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+
+class Parameter(TypedDict):
+    """A parameter record, from `status/parameters/{name}`"""
```

Comment:
If I am not mistaken we don't expose (by now) the single parameter endpoint (https://firecrest-api.cscs.ch/#/Status/get_status_parameters). 
Is this thought for future use?

Reply:
Yes, in case in the future we decide to expose more parameters for each microservice to the users.